core-sitemaps.php

```diff
@@ -25,51 +25,56 @@
  * Version:           0.3.0
  */
 
+// Bail early if Sitemaps is already defined. Prevents plugin from loading after core merge.
+if ( class_exists( 'Sitemaps' ) ) {
+	return;
+}
+
 // The limit for how many sitemaps to include in an index.
-const CORE_SITEMAPS_MAX_SITEMAPS    = 50000;
-const CORE_SITEMAPS_REWRITE_VERSION = '2020-04-29';
+const SITEMAPS_MAX_SITEMAPS    = 50000;
+const SITEMAPS_REWRITE_VERSION = '2020-04-29';
```

Comment:
All constants should be `WP_SITEMAPS_*` per core conventions I think.

Reply:
👍